src/utils/transform.js

```diff
@@ -193,6 +194,12 @@ export const getRSQuery = (componentId, props, execute = true) => {
 					execute: true,
 				  }
 				: {}),
+			...(queryType !== queryTypes.suggestion
+				? {
+					   vectorDataField: props.vectorDataField,
+					   imageValue: props.imageValue,
```

Comment:
@SavvyShah Are we making sure that these values don't get passed to the backend if not in use, for instance, if the user is not even using `imageSearch` then we need not pass it, and `vectorDataField` might not be used, in that case, we shouldn't pass it irrespective of whether the backend can handle empty values or not.

Reply:
OK